responsive-web-design/Build-product-landing-page/creative.css

```diff
@@ -0,0 +1,311 @@
+* {
+    margin: 0;
+    box-sizing: border-box;
+    text-decoration: none;
+}
+
+body {
+    /* background-color: #fda635; */
+    background-image: linear-gradient( 115deg, #ed4b4bba, #285da2a6, #834343), url(https://thewindow.barneys.com/wp-content/uploads/2017/11/4600_readyforretouching.jpg);
+    font-family: 'Open Sans', sans-serif;
+    color: black;
+    width: -webkit-fill-available;
+    height: fit-content;
+}
+#page-wrapper{
+    position: relative;
+}
+
+#header{
+    display: flex;
+    align-items: center;
+    position: fixed;
+    width: 100%;
+    /* box-shadow: 5px 5px 5px #de2222; */
+    background-color: #f11a1ae6;
+    /* background-image: linear-gradient(45deg, black, transparent); */
+}
+
+.logo{
+    display: inline-block;
+    padding: 10px;
+    text-align: center;
+}
+
+#header-img{    
+display: flex;    
+width: 150px;    
+height: 65px;
+}
+#header > nav > ul > li {
+    display: inline-block;
+    text-align: right;
+    padding: 1px 10px 10px 10px;
+    cursor: pointer;
+    margin-right: 30px;
+    font-size: large;
+    font-weight: 500;
+    /* color: cyan; */
+}
+
+#form-elements{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    height: 295px;
+    margin: 0px;
+}
+
+
+#nav-bar{
+    display: flex;
+    justify-content: flex-end;
+     width: inherit;
+    }
+
+
+.container{
+    max-width: 1000px;
+    width: 100%;
+    margin: 0 auto;
+}
+
+#features{
+    margin-top: -40px;
+    /* display: flex; */
+    /* flex-direction: column; */
+}
+
+.grid{
+    display: flex;
+}
+
+.grid > .icon > .material-icons{
+    display: flex;
+    align-items: center;
+    font-size: 47px;
+    justify-content: center;
+    height: 84px;
+    width: 12vw;
+    color: cyan;
+}
+
+.grid .men-shoes{
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    height: 109px;
+    padding: 5px;
+    font-weight: 700;
+}
+
+.grid .women-shoes{
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    height: 109px;
+    padding: 5px;
+    font-weight: 700;
+}
+
+.grid .custom-design{
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
```

Comment:
I see you are repeating the same property `justify-content` and` flex-direction` and different selectors, can you verify if you can include these properties in the parent element?

I think that if you do this the child elements going to inherit the parent property

Reply:
What happen here is that the parent element contains two child elements with different properties in regards to their position, size, margin among others. 

What I'll do is to rename the classes for the three elements to the right of the icon elements with the name(right-element), Which means that I'll have to remove two css that were unnecessary. 

Thanks for this observation.